railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -34,7 +34,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   <%- if depends_on_system_test? -%>
   # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '~> 2.13.0'
+  gem 'capybara', '~> 2.13'
```

Comment:
@jeremy 2.13.0 was set as the minimum because it added a number of minitest assertions to Capybara.  The more permissive ~>2.13 should be fine though